Cannot build at riscv64

https://buildd.debian.org/status/fetch.php?pkg=sigscheme&arch=riscv64&ver=0.9.0-1&stamp=1525954584&raw=0

```
In file included from ./include/private/gc_priv.h:68:0,
                 from mark.c:30:
./include/private/gcconfig.h:492:5: error: #error "The collector has not been ported to this machine/OS combination."
 #   error "The collector has not been ported to this machine/OS combination."
     ^~~~~
```

it may be needed to import RISC-V support from Bohem-GC ( [Add initial RISC-V support](https://github.com/ivmai/bdwgc/commit/4f7f0eebd24dcde9f2b3ec2cb98913fc39bbdda3) and [Merge RISCV 32/64-bit configurations definition](https://github.com/ivmai/bdwgc/commit/3b008f79ee29dbd0d61cf163d20eee21412df95b) ).

--- a/include/private/gcconfig.h
+++ b/include/private/gcconfig.h
@@ -479,6 +479,10 @@
 #   define NONSTOP
 #   define mach_type_known
 # endif
+# if defined(__riscv) && defined(LINUX)
+#   define RISCV
+#   define mach_type_known
+# endif
 
 /* Feel free to add more clauses here */
 
@@ -1318,6 +1322,19 @@
 #      define NO_PTHREAD_TRYLOCK
 #   endif /* DARWIN */
 # endif
+
+# ifdef RISCV
+#   define MACH_TYPE "RISC-V"
+#   define CPP_WORDSZ __riscv_xlen /* 32 or 64 */
+#   define ALIGNMENT (CPP_WORDSZ/8)
+#   ifdef LINUX
+#     define OS_TYPE "LINUX"
+      extern int __data_start[];
+#     define DATASTART ((ptr_t)__data_start)
+#     define LINUX_STACKBOTTOM
+#     define DYNAMIC_LOADING
+#   endif
+# endif /* RISCV */
 
 # ifdef NS32K
 #   define MACH_TYPE "NS32K"

